### ISSUE TITLE
Duplicate functionality between interface and trigger unit files

### ISSUE THREAD

**Issue by ssfivy** [exclave/exclave#6]
According to documentation, currently the features of the interface unit file is a superset of the trigger unit file.
Most functionality that the trigger can do can also be performed by an interface file (with the exception of STOP)
Would it be simpler to merge trigger into interfaces?

**Comment by xobs** [comment]
You're right, it might make sense to merge triggers into interfaces.  The biggest difference is that triggers are read-only and have no stdin.

In practice, this distinction is not super important, so I'll work to merge triggers into interfaces.
